spec/controllers/categories_controller_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe CategoriesController, type: :controller do
+  let(:seller) { Seller.create(username: "user1", email: "email1@email.com", password_digest: "password1") }
```

Comment:
Whenever I've tried this, `password_digest` hasn't actually worked -- I've had to type `password` and `password_confirmation`.